### PR TITLE
added msw setup for browser (#149)

### PR DESCRIPTION
**Pull request by isVivek99**
### What is the change?
added `msw` setup for browser 

### \*Dev Tested?

- [x] Yes

#### Platforms

- [x] Web

#### Browsers

- [x] Chrome

### Before / After Change Screenshots
Screenshot 2023-04-28 at 9 38 40 AM

### CODE REVIEW

File: public/mockServiceWorker.js [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+/* tslint:disable */
+
+/**
+ * Mock Service Worker (1.2.1).
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
+ */
+
+const INTEGRITY_CHECKSUM = '3d6b9f06410d179a7f7404d4bf4c3c70'
```
Comment (harshith-venkatesh) [inline public/mockServiceWorker.js:11]: Is it ok to keep this unique ID this way?

Comment (isVivek99) [reply public/mockServiceWorker.js:11]: this file is generated by `msw` when we run the command `msw-init`, i don't think it has anything to do with privacy as `msw` will only be used for development and testing purposes

File: public/mockServiceWorker.js [hunk 2]
```diff
@@ -0,0 +1,303 @@
+/* eslint-disable */
+/* tslint:disable */
+
+/**
+ * Mock Service Worker (1.2.1).
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
```
Comment (iRohitGaur) [inline public/mockServiceWorker.js:8]: need to check how to not serve this in production then

Comment (isVivek99) [reply public/mockServiceWorker.js:8]: can we check this issue, we can follow this https://github.com/Real-Dev-Squad/website-calendar/pull/150 ?

File: public/mockServiceWorker.js (hunk 1 above)
Comment (isVivek99) [reply public/mockServiceWorker.js:11]: update, we are also removing the file now, after the build has been created.

File: app/mocks/browser.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// src/mocks/browser.js
+import { setupWorker } from 'msw';
+import { handlers } from './handlers/handlers';
+
+// This configures a Service Worker with the given request handlers.
+console.log({ handlers });
```
Comment (DeRaowl) [inline app/mocks/browser.ts:6]: Dead Code

File: package.json [hunk 4]
```diff
@@ -78,6 +80,7 @@
     "husky": "^8.0.1",
     "jest": "^29.3.1",
     "jest-environment-jsdom": "^29.3.1",
+    "msw": "^1.2.1",
```
Comment (DeRaowl) [inline package.json:83]: Do we need to update for all the minor and major versions.
use of ```^```?

**Review by DeRaowl — APPROVED** [review]

**Comment by harshith-venkatesh** [comment]
Thanks for the quick setup @isVivek99 , please remove the logs so that I can approve and merge the PR

**Review by iRohitGaur — APPROVED** [review]

File: app/mocks/browser.ts (hunk 3 above)
Comment (isVivek99) [reply app/mocks/browser.ts]: removed

**Comment by iRohitGaur** [comment]
@harshith-venkatesh @isVivek99  We will have to revert this merge. Just checked. All the tests are failing

**Pull request merged**